src/styles/data-table.module.scss

```diff
@@ -50,6 +50,11 @@ table {
     padding: unset;
   }
 
+  tbody{
+    overflow: scroll;
```

Comment:
Shouldn't this be `overflow: auto`

Reply:
Instead of calculating the height, the position for the header can be put to sticky and top set to 0 with a higher z index in the tabs.module.scss file.
`.tabs-container {
  &__header {
    position: sticky;
    background-color: var(--section-background-1);
    z-index: 10;
    top: 0px;`

Reply:
no @nada-deriv ..i tried it..its not working